pyscal/factory.py

```diff
@@ -755,6 +754,13 @@ def load_relperm_df(inputfile, sheet_name=None):
             "SATNUM " + input_df["SATNUM"].astype(str) + " " + input_df["TAG"]
         )
 
+        # Check if fast is defined as a column
+        if "fast" in input_df:
+            logger.warning("Fast mode is not an option for individual SATNUMs")
+            logger.warning("it is implemented as a global option.")
+            logger.warning("The fast column in the dataframe will be ignored.")
+            logger.warning("Use fast=True in the function call instead.")
```

Comment:
Command line client users will see this if they expore possibilities for setting fast mode. Say in the last line that fast mode is only available through Python API?